releasenotes/notes/11130.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: release-notes/v2
+kind: feature
+area: traffic-management
+
+# issue is a list of GitHub issues resolved in this note.
+issue:
+  - https://github.com/istio/istio/issues/11130
+  - 11130
+
+releaseNotes: |
+  *Added* config option `values.global.proxy.holdApplicationUntilProxyStarts`,
+  which causes the sidecar injector to inject the sidecar at the start of the
+  pod's container list and configures it to block the start of all other
+  containers until the proxy is ready
+
+upgradeNotes:
+  - title: Impact on application start time
```

Comment:
This will render like: https://istio.io/latest/news/releases/1.6.x/announcing-1.6/upgrade-notes/#removal-of-legacy-helm-charts

# Impact on Start-Time
 Enabling this option affects the start up time of the pod's application 
 container(s), as its start is delayed until the proxy starts up fully.

# Impact on kubectl exec when container isn't specified
Enabling this option affects the operation of the `kubectl exec` command
when the container is not specified using the `--container` or `-c` option.
`kubectl exec` defaults to the first container in the pod. Previously,
this was the application container, so `kubectl exec` executed the command
in the application container. With this option enabled, the Istio Proxy
container becomes the first container, so using `kubectl exec` without the
`-c` option will execute the command in this instead of the application
container.

Reply:
This change doesn't break backwards compatibility with 1.6 does it? If not, I'm not sure it needs an upgrade note at all.

Reply:
I've removed upgradeNotes now.